src/features/Travel/Main.tsx

```diff
@@ -42,25 +42,38 @@ class TravelContainer extends React.Component<{}, ITravelState> {
             You are taking a bus. We should put more info about the bus here.
           </div>;
           break;
+        case 'Policy':
+          reimbursement = <div>
+            Your travel reimbursement decision has been released. In order to see how much you will be reimbursed, you must first
+            agree to our <a href="https://docs.google.com/document/d/1K8WSGQtWfKrybT_O9WrxIp93dETrv3jhy71fkHKZwdM" target="_blank">travel policy</a>.
+            <div style={{ textAlign: 'center', marginTop: '18px' }}>
+              <Button>I agree to McHacks Travel Policy</Button>
```

Comment:
We can remove this from dashboard. I added this to the reimbursement form instead